src/tribler-core/tribler_core/modules/metadata_store/orm_bindings/channel_metadata.py

```diff
@@ -85,20 +87,25 @@ def entries_to_chunk(metadata_list, chunk_size, start_index=0):
             blob = c.compress(metadata.serialized_delete() if metadata.status == TODELETE else metadata.serialized())
             # Chunk size limit reached?
             if offset + len(blob) > (chunk_size - LZ4_END_MARK_SIZE):
+                # Special case: if the blob size is bigger than the chunk size, put the blob into a separate chunk.
+                # This lets higher levels to decide what to do in this case, e.g. send it through EVA protocol.
+                if last_entry_index == start_index - 1:
```

Comment:
`last_entry_index` starts is set to `start_index - 1` by default. `start_index` is never decreased (unfortunately, we don't have `const` in Python :disappointed: ) Also, even in case the condition triggers, that will not produce any fatal consequences, but just inefficient placement of data into a separate blob.

So, nothing to worry about here :wink: